core/invoker/src/main/scala/org/apache/openwhisk/core/invoker/FPCInvokerReactive.scala

```diff
@@ -376,34 +376,21 @@ class FPCInvokerReactive(config: WhiskConfig,
   override def enable(): Route = {
     invokerHealthManager ! Enable
     pool ! Enable
-    // re-enable consumer
-    if (consumer.isEmpty)
-      consumer = Some(
-        new ContainerMessageConsumer(
-          instance,
-          pool,
-          entityStore,
-          cfg,
-          msgProvider,
-          longPollDuration = 1.second,
-          maxPeek,
-          sendAckToScheduler))
     warmUp()
     complete("Success enable invoker")
   }
 
   override def disable(): Route = {
     invokerHealthManager ! GracefulShutdown
     pool ! GracefulShutdown
-    consumer.foreach(_.close())
-    consumer = None
     warmUpWatcher.foreach(_.close())
     warmUpWatcher = None
+    consumer.foreach(_.close())
```

Comment:
Can we remove the logic : `consumer.foreach(_.close())` ?

If added, i think there may exist some message in kafka during zerodowntime deployment